docs/4.3/production.md

```diff
@@ -85,6 +85,7 @@ You can start Teleport via systemd unit by enabling the `.service` file
 with the `systemctl` tool.
 
 ```bash
+# sudo cp teleport/examples/systemd/teleport.service /etc/systemd/system
```

Comment:
should this be here?
how about just a github link to the example service

Reply:
I added this as a hint that's it's a possible, since we bundle examples into the `.tar.gz` 

I know it's not idea, but it was easier for the hand rolled setup.